manifest.go

```diff
@@ -479,7 +495,7 @@ func (m *Manifest) toRaw() rawManifest {
 	}
 	sort.Sort(sortedRawProjects(raw.Overrides))
 
-	// TODO(ibrasho): write out prune options.
+	raw.PruneOptions = toRawPruneOptions(m.PruneOptions.PruneOptions)
```

Comment:
I would recommend passing `m.PruneOptions` instead. So we can check if the projects are empty or not in `toRawPruneOptions`.